mixer/test/client/stress/stress_test.go

```diff
@@ -21,6 +21,8 @@ import (
 	"testing"
 	"time"
 
+	"os"
```

Comment:
this is a fmt issue. i don't know why the linters aren't checking this, but please run `bin/fmt.sh` all files in this PR.

Reply:
did by mistake on all files and now I got a bunch of changes :(